gammapy/image/utils.py

```diff
@@ -387,7 +387,7 @@ def coordinates(image, world=True, lon_sym=True, radians=False):
         y, x = np.indices(image.data.shape, dtype='int32') + 1
 
     wcs = WCS(image.header)
-    lon, lat = wcs.wcs_pix2world(x, y, 1)
+    lon, lat = wcs.wcs_pix2world(x, y, 0)
```

Comment:
I think that `origin=1` was correct here, because above we use `np.indices(...) + 1`.
Probably the `+1` above should be removed and `origin=0` used here, to make it more clear that we use the normal Astropy pixel coordinate convention.

But the first thing to do is to write tests for this central function in Gammapy.
E.g. one could write images to FITS and check results by zomming in to the pixel in the lower-left corner in ds9.

@mapazarr – Let me know if this is something you want to continue or if I should take over. I'd guess that adding tests for this (and maybe fixing bugs in this function or others that are discovered) can easily become quite a bit of work.